pages/news.py

```diff
@@ -16,6 +16,119 @@ def markdown_link(self):
+the HTL is decremented probabilistically on the first few hops.  In
+their tracking efforts, the Missouri police department assumes that
+when multiple requests for pieces of a file arrive from a single node,
+and all these requests have HTL 18 with no requests at HTL 17 or 16,
+this node must be the originator of the requests, with a false
+positive probability decreasing with the number of requests received
+on from the node.  This would be true if whether the HTL is
+decremented is decided per request, which has historically been the
+case in the early days of Freenet.  The vulnerability they claim to
+exploit has however been addressed in 2008 (last touched in [this
+commit][commithtl18]).  Whether the HTL is decremented is since
+decided per connection, so any probability they claim using the
+aforementioned method is false.
+
+For every connection there is a 50% chance that all the HTL18 requests
+(not only a single one, as would be the case with per-request
+probabilistic decrementing) did not originate from the node from which
+we received them but were forwarded one step. So for 10 connections
+(the lowest number of peers Freenet uses), there are on average 5
+other nodes whose HTL18 requests are forwarded with HTL18, so the
+probability that a given HTL18 request originated at the node from
+which we received it is only about 17% (1 in 6). If the node has more
+connections (normal are about 30), this probability is even lower. And
+this probability does not get higher when gathering more requests of
+chunks from a specific file or a specific kind of files, because they
+can reasonably all be forwarded from a different node — the one which
+really sent them.
+
+To get good statistics, an attacker would have to connect to the node
+under investigation over and over again at different times when the
+peers of the node changed.  This requires waiting at least 2 hours to
+change a significant number of peers — the only way to be sure would
+be to wait for the other node to go offline for more than 5 minutes
+and then to connect to it again. Then they would have to show that the
+node keeps requesting chunks from the same set of files. This could
+prove pretty hard, since downloading a 100MiB file on Freenet takes
+roughly 2 hours on a DSL connection. Screening out every node which
+ever sent a HTL17 or HTL16 request to the files they track could
+improve the reliability a lot, but with significant cost — and a
+significant rate of false negatives.
+
+If the node they track has friend-to-friend connections, even waiting
+will not allow them to find out whether the requests came from the
+node they are connected to or from the nodes connected over
+friend-to-friend connections.
+
+For details, see [the code][codehtl18].
+
+This does not make it impossible to track Freenet users who use pure
+Opennet mode (connecting to strangers), since a network where nodes
+connect to strangers is inherently susceptible to Sybil attacks.
```

Comment:
Maybe elaborate:

"Sybil attacks, where nodes of typical users are outnumbered by malicious, colluding nodes."

?